docker/docker.bzl

```diff
@@ -24,10 +24,8 @@ load(
     _docker_layer = "container_layer",
     _docker_load = "container_load",
     _docker_pull = "container_pull",
-    _docker_repositories = "repositories",
 )
 
-docker_repositories = _docker_repositories
```

Comment:
That doesn't seem to work. If we keep this, then people would be able to do something like:

```
load(
  "@io_bazel_rules_docker//docker:docker.bzl",
  "docker_repositories", "docker_pull"
)
docker_repositories()

docker_pull(
  name = "pause",
  registry = "gcr.io",
  repository = "google-containers/pause",
  tag = "2.0",
)
```
This would throw the same cyclic dependency issue.